cloud/amazon/iam_policy.py

```diff
@@ -283,7 +283,7 @@ def main():
       iam_name=dict(default=None, required=False),
       policy_name=dict(default=None, required=True),
       policy_document=dict(default=None, required=False),
-      policy_json=dict(default=None, required=False),
+      policy_json=dict(type='dict', default=None, required=False),
```

Comment:
@bcoca type='json' doesn't work for my json in 2.2.  

My json
```
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Sid": "Stmt1486673523000",
      "Effect": "Allow",
      "Action": [
        "swf:*"
      ],
      "Resource": [
        "arn:aws:swf:*:763396238289:/domain/qa"
      ]
    }
  ]
}
```

How I use it in the module:
```
policy_json: "{{ lookup( 'template', 'swf-policy.json.j2') }}"
```

Gives me "Syntax errors" response.  

If I change to `dict` it works.